assembl/static2/js/app/components/debate/common/sentiments.jsx

```diff
@@ -50,60 +51,62 @@ export const Sentiment = ({
           );
           displayModal(null, body, true, null, null, true);
         } else {
-          client.mutate(
-            isSelected
-              ? {
-                mutation: deleteSentimentMutation,
-                variables: { postId: postId },
-                optimisticResponse: {
-                  deleteSentiment: {
-                    post: {
-                      id: postId,
-                      sentimentCounts: {
-                        like: likeCount - (mySentiment === sentiment.type ? 1 : 0),
-                        disagree: disagreeCount - (mySentiment === sentiment.type ? 1 : 0),
-                        dontUnderstand: dontUnderstandCount - (mySentiment === sentiment.type ? 1 : 0),
-                        moreInfo: moreInfoCount - (mySentiment === sentiment.type ? 1 : 0),
-                        __typename: 'SentimentCounts'
+          client
+            .mutate(
+              isSelected
+                ? {
+                  mutation: deleteSentimentMutation,
+                  variables: { postId: postId },
+                  optimisticResponse: {
+                    deleteSentiment: {
+                      post: {
+                        id: postId,
+                        sentimentCounts: {
+                          like: likeCount - (mySentiment === sentiment.type ? 1 : 0),
```

Comment:
??? oups, you decrement all sentiments here. 
please add `const isMySentiment = mySentiment === sentiment.type` then for each sentiment use this condition `isMySentiment && sentiment.type == SENTIMENTS.like ? 1: 0` for the like sentiment ....
you need to add the SENTIMENTS global const if not exists

Reply:
I didn't do change anything here apart from the indentation, this code already existed and was working properly so far, otherwise you would have seen it in threads

Reply:
In addition, I personnaly prefer the way it was written, less code, exact same logic and avoid the need to compare to a global constant as we compare values retrieved by the backend

Reply:
I didn't change this part of the code, only indent. It seems the request is so fast that maybe nobody have seen this bug. I ll fix it.